### PR TITLE
Feature: Add highlight for minos inquisitors to make them easier to see

### PR DESCRIPTION
**Pull request by brandonwamboldt**

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/features/event/diana/HighlightInquisitors.kt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import net.minecraft.entity.Entity
+import net.minecraft.entity.player.EntityPlayer
+import net.minecraftforge.event.entity.EntityJoinWorldEvent
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class HighlightInquisitors {
+    private val config get() = SkyHanniMod.feature.event.diana
+
+    @SubscribeEvent
+    fun onEntityHealthUpdate(event: EntityMaxHealthUpdateEvent) {
```
Comment (Thunderblade73) [inline src/main/java/at/hannibal2/skyhanni/features/event/diana/HighlightInquisitors.kt]: question, why do you need this event. EntityJoinWorld should be fine to get the entity.

Comment (brandonwamboldt) [reply src/main/java/at/hannibal2/skyhanni/features/event/diana/HighlightInquisitors.kt]: If you spawn the entity it joins before receiving a name sometimes, so it gets caught by this. If other players spawn the entity, and you just get close enough to load it, the join world event catches it. Overall i've been pondering a rework for how mobs are detected.

Comment (hannibal002) [reply src/main/java/at/hannibal2/skyhanni/features/event/diana/HighlightInquisitors.kt]: @brandonwamboldt  There is already a big open pr that deals with mob spawn and despawn detection

Also this pr could be the last feature pr before 0.22

Comment (Thunderblade73) [reply src/main/java/at/hannibal2/skyhanni/features/event/diana/HighlightInquisitors.kt]: Player Entitys are should always spawn with the name only other mobs are a problem. And for the mob detection part, I have my [PR#712](https://github.com/hannibal002/SkyHanni/pull/712) for that.

Comment (brandonwamboldt) [reply src/main/java/at/hannibal2/skyhanni/features/event/diana/HighlightInquisitors.kt]: Removed the max health update code 👍

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**